templates/part.boardlist.php

```diff
@@ -3,7 +3,7 @@
 		<thead>
 		<tr>
 			<td class="cell-board-bullet"></td>
-			<td class="cell-board-title" width="90%"><?php p($l->t('Board title')); ?></td>
+			<td class="cell-board-title" width="90%"><?php p($l->t('Boards')); ?></td>
```

Comment:
`Title` is better.

By the way, Files app uses `Name` for the same row in their table